mono/mini/aot-compiler.c

```diff
@@ -6332,7 +6341,10 @@ encode_patch (MonoAotCompile *acfg, MonoJumpInfo *patch_info, guint8 *buf, guint
 	case MONO_PATCH_INFO_AOT_JIT_INFO:
 	case MONO_PATCH_INFO_GET_TLS_TRAMP:
 	case MONO_PATCH_INFO_SET_TLS_TRAMP:
+	case MONO_PATCH_INFO_TRAMPOLINE_FUNC_ADDR:
 		encode_value (patch_info->data.index, p, &p);
```

Comment:
missing break?

Reply:
It does look like it on web page. I’ll double check.

Reply:
Agreed, and fixed. Thank you. Test coverage, hm?